pkgs/development/mobile/androidenv/androidsdk.nix

```diff
@@ -78,6 +78,8 @@ stdenv.mkDerivation rec {
       ${stdenv.lib.optionalString (stdenv.hostPlatform.system == "x86_64-linux") ''
         for i in emulator emulator-check
         do
+            patchelf --set-interpreter ${stdenv.cc.libc.out}/lib/ld-linux-x86-64.so.2 $i
+            patchelf --set-rpath ${stdenv.cc.cc.lib}/lib64 $i
```

Comment:
Was is not possible to set the rpath and interpreter within the same invocation?

Reply:
This also seems to duplicated `LD_LIBRARY_PATH` below. Is `emulator` calling any unpatched binaries or is using `dlopen` at runtime that it needs LD_LIBRARY_PATH? Otherwise all libraries could be moved up to this rpath invocation.

Reply:
> This also seems to duplicated `LD_LIBRARY_PATH` below. Is `emulator` calling any unpatched binaries or is using `dlopen` at runtime that justify LD_LIBRARY_PATH? Otherwise all libraries could be moved up to this rpath invocation.

I can't answer that.

> Was is not possible to set the rpath and interpreter within the same invocation?

I combined the invocation.